rsmtool/utils/metrics.py

```diff
@@ -210,40 +217,36 @@ def standardized_mean_difference(
 
     Parameters
     ----------
-    y_true_observed : array-like
+    y_true_observed : numpy.ndarray
         The observed scores for the group or subgroup.
-    y_pred : array-like
-        The predicted score for the group or subgroup.
-    population_y_true_observed_sd : float, optional
-        The population true score standard deviation.
-        When the SMD is being calculated for a subgroup,
-        this should be the standard deviation for the whole
-        population.
+    y_pred : numpy.ndarray
+        The predicted scores for the group or subgroup.
+    population_y_true_observed_sd : Optional[float]
+        The population true score standard deviation. When the SMD is being
```

Comment:
Should it be `SD`(standard deviation) instead of `SMD`?

Reply:
No, SMD is standardized mean difference which is what this function is computing.